deployment/README.md

```diff
@@ -2,6 +2,8 @@ Dockerfile
 ===
 ## 1.Description  
 This is the Dockerfile of nni project, including the most kinds of deeplearning frameworks and nni source code.  You can run your nni experiment in this docker container directly.
+Dockerfile.build.base is the base Docker, including Ubuntu, cudnn and the NNI environment.
+Dockerfile is the customized docker for users, if you want to add your own deeplearning environment, you could update this Dockerfile.
```

Comment:
this line is not accurate, Dockerfile is a text document that contains all the commands a user could call on the command line to assemble an image.

Reply:
updated.